src/service/fixed-layer.js

```diff
@@ -125,7 +126,7 @@ export class FixedLayer {
     this.trySetupSelectorsNoInline(fixedSelectors, stickySelectors);
 
     // Sort in document order.
-    this.sortInDomOrder_(this.elements_);
+    this.sortInDomOrder_();
```

Comment:
Nit: I liked seeing what's being sorted. Update comment at least?

Reply:
There's no need to pass instance variables to an instance function.

Reply:
Just a readability nit. I.e. what are we sorting? All elements? Some elements? Something else?